website/docs/r/service_networking_application_load_balancer.html.markdown

```diff
@@ -0,0 +1,62 @@
+---
+subcategory: "Service Networking"
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_application_load_balancer"
+description: |-
+  Manages an Application Gateway for Containers (ALB).
+---
+
+# azurerm_alb
+
+Manages an Application Gateway for Containers (ALB).
+
+## Example Usage
+
+```hcl
+resource "azurerm_application_load_balancer" "example" {
+  name                = "example"
+  resource_group_name = "example"
+  location            = "West Europe"
+}
+```
+
+## Arguments Reference
+
+The following arguments are supported:
+
+* `name` - (Required) The name which should be used for this resource. Changing this forces a new resource to be created.
+
+* `resource_group_name` - (Required) The name of the Resource Group where the resource should exist. Changing this forces a new resource to be created.
+
+* `location` - (Required) The Azure Region where the resource should exist. Changing this forces a new resource to be created.
+
+**Note:** The available values of `location` are `northeurope` and `north central us`.
```

Comment:
The list of supported locations/regions may be found here: https://learn.microsoft.com/azure/application-gateway/for-containers/overview#supported-regions